### ISSUE TITLE
Failed reading SVG, config should be an object

### ISSUE THREAD

**Issue by bartosjiri** [poppa/sveltekit-svg#8]
## Problem 
Plugin options are specified as optional in the README, however not providing any `svgoOptions` causes following error when running `svelte-kit build` and importing SVG as a Svelte component:

```
Failed reading SVG "C:/Users/[redacted]/static/[redacted].svg": Config should be an object
```
Then when running `svelte-kit dev`, a subsequent error is produced as expected:
```
Error: <Icon> is not a valid SSR component. You may need to review your build config to ensure that dependencies are compiled, rather than imported as pre-compiled modules
```

## Workaround
Providing empty object for `svgoOptions` solves mentioned errors:
#### svelte.config.js
```
const svg = require('@poppanator/sveltekit-svg')

module.exports = {
  ...,
  kit: {
    ...,
    vite: {
      plugins: [svg({svgoOptions: {}})]
    }
  }
}
```

### Dependencies
- `@poppanator/sveltekit-svg@0.1.4`
- `@sveltejs/kit@1.0.0-next.166`

#### svelte.config.js
```
const svg = require('@poppanator/sveltekit-svg')

module.exports = {
  ...,
  kit: {
    ...,
    vite: {
      plugins: [svg()],
      resolve: {
        alias: {
          $assets: resolve('./static/assets'),
        },
      }
    }
  }
}
```

#### src/routes/index.svelte
```
<script>
  import Icon from '$assets/icon.svg';
</script>

<Icon />
```

#### static/assets/icon.svg
`-`

**Comment by poppa** [comment]
Thanks for the bug report 👍

After a quick glance there seems to be a couple of issues at hand: There seems to have been some regression in `svgo` which lead to an error when passing `undefined` and not `null` as options argument.

But once I fixed that there seems to have been some regression with the Svelte/Vite/Rollup plugin API. I now get an error saying _'claim_svg_element' is not exported by ../node_modules/svelte/internal/index.mjs_. I yet don't know if this is due to a change in the plugin API or if there has snuck in a bug in the Svelte compiler itself.

Anyhow, I'm looking into it and will get back once I've figured out what is causing the error.
